CMakeLists.txt

```diff
@@ -262,6 +262,14 @@ set_target_properties(datachannel PROPERTIES
 	SOVERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}
 	CXX_STANDARD 17
 	CXX_VISIBILITY_PRESET default)
+
+if(APPLE)
+	set_target_properties(datachannel PROPERTIES
+		SOVERSION ${PROJECT_VERSION}
+		MACHO_COMPATIBILITY_VERSION 1.0.0
+		MACHO_CURRENT_VERSION ${PROJECT_VERSION})
```

Comment:
`MACHO_COMPATIBILITY_VERSION` and `MACHO_CURRENT_VERSION` were introduced in CMake 3.17, so you should bump `cmake_minimum_required` to 3.17.